mvn_setup/tasks-parent/tasks-ui/src/taskclient/client.ts

```diff
@@ -16,27 +16,32 @@ const mockUsers: User[] = [
   {
     displayName: "Carrot Ironfoundersson",
     userId: "carrot ironfoundersson",
-    userRoles: ["admin-role"]
+    userRoles: ["admin-role"],
+    avatar: "CI"
   },
   {
     displayName: "Sam Vimes",
     userId: "sam vimes",
-    userRoles: ["admin-role"]
+    userRoles: ["admin-role"],
+    avatar: "SV"
   },
   {
     displayName: "Lord Vetinari",
     userId: "lord vetinari",
-    userRoles: ["admin-role"]
+    userRoles: ["admin-role"],
+    avatar: "LV"
   },
   {
     displayName: "Lady Sybil Vimes",
     userId: "lady sybil vimes",
-    userRoles: ["admin-role"]
+    userRoles: ["admin-role"],
+    avatar: "LV"
```

Comment:
Let's change this avatar code to "LS", even though it's not technically this person's last name.  But since this is just for testing, it's ok, and it would help if the avatar is different.